src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -240,7 +241,14 @@ export class NotebookMarkdownRenderer {
 		} else if (href.charAt(0) === '/') {
 			return base.replace(/(:\/*[^/]*)[\s\S]*/, '$1') + href;
 		} else if (href.slice(0, 2) === '..') {
-			return path.join(base, href);
+			if (isWindows) {
+				// we need to format invalid href formats (ex. ....\file to ..\..\file)
```

Comment:
Why is this invalid only on windows?

Reply:
The href we get back when testing on mac all seem to properly formatted and that is why we can do a join properly. The href after it gets parsed from marked.js on windows gets back into the format stated in the example and as result will always resolve incorrectly.

Reply:
That seems not great....would probably be worth digging into why that's happening since that sounds like a possible bug there and we should make sure it's not going to affect anything else.

Reply:
Yeah discussed this with Barbara as well, we both saw that it was coming via the HTML/marked js component that is being sent and therefore was not able to change it there. We plan on doing a bug bash after this release on linking in general as there are many different entry points, this change was targetting specifically WYSIWYG and resolving the link on windows.